owl/test/swrl.owl

```diff
@@ -322,7 +345,7 @@
 
     <owl:NamedIndividual rdf:about="http://knowrob.org/kb/swrl_test#RectangleBig">
         <rdf:type rdf:resource="http://knowrob.org/kb/swrl_test#Rectangle"/>
-        <hasHeightInMeters rdf:datatype="http://www.w3.org/2001/XMLSchema#integer">13</hasHeightInMeters>
+        <hasHeightInMeters rdf:datatype="http://www.w3.org/2001/XMLSchema#float">13.5</hasHeightInMeters>
```

Comment:
It might be not ok to change this. Because I have a unit-test that checks for the result of height*width. It might be ok to add new things to ontologies, but you need to be extremely careful when changing things.

Reply:
Fixed it.